src/ui/styles/RadioButton.scss

```diff
@@ -6,7 +6,7 @@
   position: absolute;
   z-index: 0;
   opacity: 0;
-  width: 24px;
+  width: 100%;
   height: 24px;
```

Comment:
How did these `px` values slip under the radar 😅

Reply:
The entire Radio Button component seems to be styled using `px` rather than `rem`, I can chat to Dave to assert why if required.